third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -457,9 +460,64 @@ func TestAccComputeDisk_resourcePolicies(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
```

Comment:
If you reuse the block right above here, placing another import step after the config step, the provider will do an import using a blank state and verify that the state is identical to the state used to create the resource. That's sufficient to check that values are set upstream, so you can remove the explicit `Check` by doing so.

Lots of tests in this file use Checks, but we primarily use newer "import-style" tests now 🙂